src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag

```diff
@@ -0,0 +1,27 @@
+<%@ tag description="adminHome.jsp - Admin home create instructor account form" %>
+<%@ tag import="teammates.common.util.Const" %>
+
+<div class="well well-plain">
+    <form method="post" action="<%=Const.ActionURIs.ADMIN_INSTRUCTORACCOUNT_ADD%>" name="form_addinstructoraccount">
+        <div>
+            <label class="label-control">Short Name:</label>
+            <input class="form-control" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_SHORT_NAME%>" value="${data.instructorShortName}">
```

Comment:
I know it would be a bit painful but we should pass the attributes of `data` from the main .jsp file, except for `data.account`.